libs/ledgerjs/packages/hw-app-btc/src/Btc.ts

```diff
@@ -267,15 +260,4 @@ export default class Btc {
       additionals
     );
   }
-
-  async changeImplIfNecessary(): Promise<BtcOld | BtcNew> {
-    const appAndVersion = await getAppAndVersion(this._transport);
-    if (appAndVersion.name === "Exchange" && this._impl instanceof BtcNew) {
-      const isBtcLegacy = await checkIsBtcLegacy(this._transport);
-      if (isBtcLegacy) {
-        this._impl = new BtcOld(this._transport);
-      }
-    }
-    return this._impl;
-  }
```

Comment:
what's the status of this PR change vs https://github.com/LedgerHQ/ledger-live/pull/2952